src/coreclr/jit/llvm.cpp

```diff
@@ -48,6 +48,7 @@ static const CorInfoTypeWithMod (*_getArgTypeIncludingParameterized)(void*, CORI
 static const CorInfoTypeWithMod (*_getParameterType)(void*, CORINFO_CLASS_HANDLE, CORINFO_CLASS_HANDLE*);
 static const TypeDescriptor (*_getTypeDescriptor)(void*, CORINFO_CLASS_HANDLE);
 static CORINFO_METHOD_HANDLE (*_getCompilerHelpersMethodHandle)(void*, const char*, const char*);
+static const uint32_t (*_getInstanceFieldAlignment)(void*, CORINFO_CLASS_HANDLE);
```

Comment:
I presume all of `_getInstanceFieldAlignment` is not necessary anymore.

Reply:
`StructIsWrappedPrimitive` can go, is that what you meant?

Reply:
No, I meant `_getInstanceFieldAlignment`, but I see it is actually still used. So nothing needs to be done about it.